From 91c6e714c342748b32066d7966f0e5c61901f6db Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Wed, 16 May 2018 11:20:37 +0800
Subject: [PATCH 1935/2328] media: intel-ipu4: imx355: Enable vsync output

Message for Open Source:
Enable vsync output.

Message for Internal:
[Issue/Feature] Frame synchronization.
[Root Cause/Changes] Enable vsync output.

Change-Id: I3621fe10164839d248abca1780b26b82aabe3639
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/imx355.c | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/drivers/media/i2c/imx355.c b/drivers/media/i2c/imx355.c
index 7405b79..b0b5c82 100644
--- a/drivers/media/i2c/imx355.c
+++ b/drivers/media/i2c/imx355.c
@@ -149,6 +149,20 @@ static const struct imx355_reg imx355_global_regs[] = {
 	{ 0x4933, 0x05 },
 	{ 0x595e, 0x01 },
 	{ 0x5963, 0x01 },
+	{ 0x3030, 0x01 },
+	{ 0x3031, 0x01 },
+	{ 0x3045, 0x01 },
+	{ 0x4010, 0x00 },
+	{ 0x4011, 0x00 },
+	{ 0x4012, 0x00 },
+	{ 0x4013, 0x01 },
+	{ 0x68a8, 0xfe },
+	{ 0x68a9, 0xff },
+	{ 0x6888, 0x00 },
+	{ 0x6889, 0x00 },
+	{ 0x68b0, 0x00 },
+	{ 0x3058, 0x00 },
+	{ 0x305a, 0x00 },
 };
 
 static const struct imx355_reg_list imx355_global_setting = {
-- 
2.7.4

